### PR TITLE
Fix Style Check step

### PR DESCRIPTION
**Pull request by PProfizi**
Pre-commit step is currently failing due to new version of `isort` not being used. Add update command when installing pre-commit.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -30,9 +30,11 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4.5.0
         with:
-          python-version: 3.9
+          python-version: 3.10
```
Comment (PProfizi) [inline .github/workflows/ci.yml:33]:
```suggestion
          python-version: "3.10"
```

**Comment by PProfizi** [comment]
Hi @RobPasMue I am trying to fix the problem with pre-commit and isort you mentioned. I tried updating the Style Check step in our CI to match what is done in the pyansys action yet it is not enough.
Does this mean I have to pin `isort=5.12.0` somewhere? It does not appear anywhere right now and the `pip install pre-commit` step does not even seem to install `isort`.

**Comment by RobPasMue** [comment]
HI @PProfizi! Let me fix this quickly. Steps to reproduce my upcoming changes are:

```bash
pre-commit autoupdate
pre-commit run --all-files
``

The version has to be changed at the ``pre-commit-config.yaml`` file. Your Python version was fine as it was, no worries about that.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -30,9 +30,11 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4.5.0
         with:
-          python-version: 3.9
+          python-version: "3.10"
```
Comment (RobPasMue) [inline .github/workflows/ci.yml:33]:
```suggestion
          python-version: 3.9
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -30,9 +30,11 @@ jobs:
...
       - name: Install pre-commit
-        run: pip install pre-commit
+        run: |
+          python -m pip install --upgrade pip pre-commit
+          pre-commit install
```
Comment (RobPasMue) [inline .github/workflows/ci.yml:37]:
```suggestion
        run: pip install pre-commit
```

**Review by RobPasMue — APPROVED** [review]
This should work now 😄 thanks for pinging me @PProfizi

**Comment by PProfizi** [comment]
> This should work now 😄 thanks for pinging me @PProfizi

Thank you so much @RobPasMue!

**Comment by PProfizi** [comment]
@RobPasMue one question though: I thought dependabot would deal with the pre-commit.yaml dependencies, yet I guess I haven't plugged this in.

**Comment by PProfizi** [comment]
> @RobPasMue one question though: I thought dependabot would deal with the pre-commit.yaml dependencies, yet I guess I haven't plugged this in.

Sorry, just checked, and it does not support it yet I think.

**Comment by RobPasMue** [comment]
Nope, that's not supported. What we can do is set up pre-commit.ci though. This tool will automatically open pre-commit related update PRs on a weekly basis. And it will also allow you to remove the style-checks step of your workflows since it runs on cached runners always. See this issue for a summary of the benefits: https://github.com/pyansys/pyfluent/issues/1253

If you wanted it on your repository let me know and I can help you set it up 😄

**Pull request merged**